input_definition.go

```diff
@@ -372,7 +371,7 @@ func HandleAction(a Action, value interface{}, colID uint64, timestamp int64) (*
 		}
 		bit.RowID = uint64(v)
 	case InputSetTimestamp:
```

Comment:
I think we should add a comment about where the timestamp is handled earlier, and why we need to return nil.

Reply:
how about:
//  InputSetTimestamp action is used in the InputJSONDataParser Handler to append a timestamp to all bits in the frame. There are no individual rowID's to set, and the action is a no-op at this step.